edb/lib/_testmode.edgeql

```diff
@@ -359,6 +359,15 @@ std::_datetime_range_buckets(
     $$;
 };
 
+
+CREATE FUNCTION
+std::_current_settings(sqlname: str) -> OPTIONAL std::int64 {
+    USING SQL $$
+      SELECT nullif(current_setting(sqlname, true), '')::int8
+    $$;
+};
```

Comment:
Let's make it return `str` and drop the `nullif` bit. If we're going to make it generic, we should